src/timer.rs

```diff
@@ -0,0 +1,27 @@
+use std::time::{Duration, Instant};
```

Comment:
We should use the [`instant`](https://crates.io/crates/instant) crate here, because `std::time::Instant` is not supported on the web.

Reply:
Thanks for figuring that out for me :)